01_prepare_host.sh

```diff
@@ -84,8 +84,8 @@ source lib/images.sh
 
 # Add usr/local/go/bin to the PATH environment variable
 GOBINARY="/usr/local/go/bin"
-if [[ ":$PATH:" != *":$GOBINARY:"* ]]; then
-  echo export PATH="$PATH":/usr/local/go/bin >> ~/.bashrc
+if [ ":$PATH:" != *":${GOBINARY}:"* ]; then
+  echo "export PATH=${PATH}:/usr/local/go/bin" >> ~/.bashrc
   # shellcheck disable=SC1090
   source ~/.bashrc
```

Comment:
Why are we re-sourcing `.bashrc`? You could just export PATH. Also, you`re testing with `$GOBINARY`, why not have that in export rather than hard coded path?